code/Test_definitions/location-verification.feature

```diff
@@ -18,6 +18,10 @@ Feature: CAMARA Device location verification API, vwip - Operation verifyLocatio
     And the header "x-correlator" is set to a UUID value
     And the request body is set by default to a request body compliant with the schema
 
+##########################

```

Comment:
This is not related to linting, I would keep it simple and with a style aligned across CAMARA. In Commonalities artifacts examples use single line comments. Not sure about other WGs